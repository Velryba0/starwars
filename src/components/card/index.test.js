import React from 'react'
import { shallow } from 'enzyme'
import CardTable from './index'

let wrapped = shallow(<CardTable/>);

describe('src/components/card', () => {
  it('load component', () => {   
    expect(wrapped).toMatchSnapshot();
  });  
});