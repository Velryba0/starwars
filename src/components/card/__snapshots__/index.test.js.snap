// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/card load component 1`] = `ShallowWrapper {}`;
